src/config/options.js

```diff
@@ -8,11 +8,11 @@ export const options = {
     },
     heading: {
       normalized: 'Normalized Compendia',
-      rnaSeq: 'RNA-seq Sample Compendia'
+      'rna-seq': 'RNA-seq Sample Compendia'
```

Comment:
This is a temporary fix to prevent an error due to removal of the `type` prop. The `compendia` property will be removed from `config/options` in a subsequent issue (#401).